serpentTools/parsers/depletion.py

```diff
@@ -130,3 +130,13 @@ def _processChunk(self, chunk, name, variable):
                     continue
                 cleaned.append(line[:line.index('%')])
         self.materials[name].addData(variable, cleaned)
+
+    def _precheck(self):
+        """todo
+        """
+        pass
+
+    def _postcheck(self):
+        """todo
+        """
+        pass
```

Comment:
For now, check that materials have in fact been stored

Reply:
OK, I added something that counts the number of materials before running the parse in the precheck, and the postcheck verifies that number got parsed.